ocs_ci/utility/deployment.py

```diff
@@ -152,6 +152,11 @@ def get_and_apply_icsp_from_catalog(image, apply=True, insecure=False):
 
     if apply and not config.DEPLOYMENT.get("disconnected"):
         exec_cmd(f"oc apply -f {icsp_file_dest_location}")
-        wait_for_machineconfigpool_status("all")
+        managed_ibmcloud = (
+            config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM
+            and config.ENV_DATA["deployment_type"] == "managed"
+        )
+        if not managed_ibmcloud:
```

Comment:
I think that applying the ICSP will not work anyway in IBM cloud ROKS deployment. I I guess this is the W/A for also Live deployment.